src/graphx/graphx.asm

```diff
@@ -829,22 +829,21 @@ _SetDraw:
 ;  arg0: buffer or screen
 ; Returns:
 ;  None
+	pop	de
 	pop	hl
```

Comment:
This should be `ex (sp),hl` instead of `pop hl`. It is the caller's responsibility to deallocate the argument stack space, not the callee's. Callees must leave the stack space in place (although you can overwrite the data, as `ex (sp),hl` does).